recipes/r-biwt/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = '1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-biwt
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: biwt_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/biwt_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/biwt/biwt_{{ version }}.tar.gz
+
+  sha256: 2428758fad46b1569b58e4985806c0df79e9fd3a3115d4902512f5e127ea63d5
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-mass
+    - r-rrcov
+
+  run:
+    - r-base
+    - r-mass
+    - r-rrcov
+
+test:
+  commands:
+    - $R -e "library('biwt')"  # [not win]
+    - "\"%R%\" -e \"library('biwt')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=biwt
+  license: GPL-2
+  summary: Compute multivariate location, scale, and correlation estimates based on Tukey's biweight
+    M-estimator.
+  license_family: GPL2
+
+extra:
+  recipe-maintainers:
+    - MathiasHaudgaard
```

Comment:
If you like please add these guys as well: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml

Reply:
I'll do that for the following recipes :-)